.github/workflows/tidy-format-validation.yml

```diff
@@ -11,7 +11,7 @@ jobs:
         run: |
           sudo apt-get update
           sudo apt-get install -y libsdl2-dev libpng-dev libglew-dev ninja-build clang-tidy clang-format-12
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
```

Comment:
i changed this to get the "you shouldn't use node 12" warning to go away (in hopes it might fix the commenting issue, but it didn't)